.github/workflows/ci.yml

```
@@ -114,13 +114,15 @@ jobs:
     
     - name: Restore dependencies
       run: |
-        dotnet restore src/fcode.fsproj
+        dotnet restore src/fcode.fsproj --verbosity normal
+        dotnet restore tests/fcode.Tests.fsproj --verbosity normal
     
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)